src/OrleansTestingHost/TestingSiloOptions.cs

```diff
@@ -46,6 +50,8 @@ public class TestingSiloOptions
         public bool ParallelStart { get; set; }
         public GlobalConfiguration.ReminderServiceProviderType ReminderServiceType { get; set; }
         public string DataConnectionString { get; set; }
+        public Action<ClusterConfiguration> ConfigurationCustomizer { get; set; }
```

Comment:
I will create a smaller PR that has only the testing stuff in it.
(It will be a bit nonsensical since we can't actually test anything real about multiple clusters without the functionality there, but perhaps it helps in general just to reduce the size of the PR)